flink-connector-kafka/src/main/java/org/apache/flink/streaming/connectors/kafka/table/KafkaConnectorOptionsUtil.java

```diff
@@ -145,7 +145,7 @@ public static void validateSinkTopic(ReadableConfig tableOptions) {
         }
     }
 
-    private static void validateScanStartupMode(ReadableConfig tableOptions) {
```

Comment:
nit: we can keep this `private` as this PR does not add the scan startup mode to upsert kafka.

Reply:
Changed this by mistake, will revert.